netci.groovy

```diff
@@ -285,6 +285,41 @@ def osShortName = ['Windows 10': 'win10',
 }
 
 // **************************
+// Define perf testing.  These tests should be run daily, and will run on Jenkins not in Helix
+// **************************
+[true, false].each { isPR ->
```

Comment:
if it's only being run as a daily triggered build, do you still need the `isPR` switch?